### PR TITLE
add logging

### PR DESCRIPTION
**Pull request by archio1**

### CODE REVIEW

File: src/FakeClass.py [hunk 1]
```diff
@@ -16,11 +14,17 @@
 from pathlib import Path
 from pandas import Series
 from typing import Union
+import logging
 from nltk.corpus import stopwords
 nltk.download('stopwords')
 parent_path = Path().resolve()
```
Comment (merseneries) [inline src/FakeClass.py]: Add an empty line between imports and download

```python
from nltk.corpus import stopwords

nltk.download('stopwords')
parent_path = Path().resolve()
```

File: src/FakeClass.py [hunk 2]
```diff
@@ -16,11 +14,17 @@
...
 
 
+logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s',
```
Comment (merseneries) [inline src/FakeClass.py]: why do you use here `-8s` for `levelname`?

**Review by merseneries — CHANGES_REQUESTED** [review]
Three points from me:

1. Part of the code needs formatting. Install `black` package to format the code. 
2. Log messages don't have to be in every method, just the main ones.
3. Use the `present continuous` tense to write a message.

File: src/FakeClass.py (hunk 2 above)
Comment (archio1) [reply src/FakeClass.py]: because i think about better fromation.

**Comment by merseneries** [comment]
Good

**Pull request merged**